doc/api/http2.md

```diff
@@ -3163,9 +3163,9 @@ Url {
   href: '/status?name=ryan' }
 ```
 
-To extract the parameters from the query string, the
-`require('querystring').parse` function can be used, or
-`true` can be passed as the second argument to `require('url').parse`.
+To obtain the parameters from the query string, use the
+`require('querystring').parse()` function or pass
```

Comment:
Could we also make these (`require('querystring').parse()`, `require('url').parse()`) links to appropriate docs?

Reply:
If we do make them links, I'd probably prefer to simplify the text to `querystring.parse()` and `url.parse()`. Does that work for you?

Reply:
Other things to consider (not necessarily for this pull request because there might be a lot of conversation to have): Do we want to suggest WHATWG `URL` instead of `url.parse()`? Do we want to remove this paragraph entirely as out of scope?

Reply:
I'd go for both of your suggestions :+1: .
Especially considering https://github.com/nodejs/node/pull/34914 and https://github.com/nodejs/node/pull/34931.

Reply:
@lundibundi I think the first suggestion (the one you made) is definitely a worthwhile change and the second one (the one I asked about) is probably a worthwhile change as well, but I would prefer to do them in a subsequent PR as they will involve changing at least one other paragraph as well as two code examples. I'd rather keep the rewording here isolated and small.

Reply:
https://github.com/nodejs/node/pull/34956